docs/daikon/product_timeseries_metrics.md

```diff
@@ -60,3 +64,26 @@ GET
     "product_listings_rtn": 876
 }
 ```
+Note that many census division names include special characters such as spaces and accents. The unencoded names of such census divisions can be passed to a request as-is:<br>
```

Comment:
Would include notes like this at the end and smaller text

Reply:
i was thinking these 2 sentences give a little context as to why the next example query is there (i.e. it includes a non-encoded census division name with special characters) so for that reason i feel like it makes sense to preface the query with this text. maybe the "Note" lingo makes it seem too trivial? if you still feel like the context is best as a footnote kind of thing I can def do that though.

Reply:
sgtm